lib/sidekiq.rb

```diff
@@ -127,9 +129,13 @@ def self.logger=(log)
     Sidekiq::Logging.logger = log
   end
 
+  # When set, overrides Sidekiq.options[:global_poll_interval_average] and sets the
+  # average interval that this process will delay before checking for scheduled jobs
+  # or job retries that are ready to run.
+  #
   # See sidekiq/scheduled.rb for an in-depth explanation of this value
   def self.poll_interval=(interval)
```

Comment:
Understood, totally agree.  I think the encapsulation that a setter gives is useful, e.g. when you are deprecating something and want to warn the user.  :smile:

I'd prefer not to add Sidekiq::Config at this point in time.  Sidekiq has gotten pretty stable at this point; I'd rather not refactor unless there is a functional reason.